conda_environment.yml

```diff
@@ -20,3 +20,4 @@ dependencies:
   - toml=0.10.2=pyhd8ed1ab_0
   - tomopy=1.11.0=py310h8d81d44_101
   - yaml=0.2.5=h7f98852_2
+  - pooch
```

Comment:
This was the python package I remember existing but couldn't remember the name of. Thank you!